api/opds_for_distributors.py

```diff
@@ -306,12 +306,7 @@ def update_work_for_edition(self, *args, **kwargs):
         pool, work = super().update_work_for_edition(
             *args, is_open_access=False, **kwargs
         )
-        pool.update_availability(
-            new_licenses_owned=1,
-            new_licenses_available=1,
-            new_licenses_reserved=0,
-            new_patrons_in_hold_queue=0,
```

Comment:
`pool.unlimited_access = True` sets `licenses_owned` and `licenses_available`, but doesn't reset `licenses_reserved` or `patrons_in_hold_queue`. Because all OPDS for Distributor books are either open access or unlimited access, it seems like this shouldn't be a problem, since these values should never be changed from their default of zero; but, just wanted to ensure that this has been considered.

Reply:
Yeah, I left it alone. I think those arguments were only there because _something_ had to be passed in.